azureeyemodule/README.md

```diff
@@ -810,6 +810,74 @@ bool ObjectDetector::pull_data(cv::GStreamingCompiled &pipeline)
 }
 ```
 
+### Using UVC(USB Video Class) camera as input source
+
+Be default, the Azure Percept's Camera attached to Eye SoM is used as input source.
+
+We can also use a UVC camera as input source. 
+
+A generic UVC camera that outputs raw format (like RGB and YUV) is supported.
+Validated on Logitech B525 HD, 720P, 30 fps.
+
+1. Build the dockerfile and pull it as you did which was mentioned in earlier sections.
+1. Plug in your UVC camera. In SSH, run command *sudo ls /dev/video**. You should see at least one device (maybe 2 or even more). By default it should be video0.
+1. Now run the Docker container like this:
+    ```
+    docker run --rm \
+               -v /dev/bus/usb:/dev/bus/usb \
+               --device=/dev/video0 \
+               -it \
+               --device-cgroup-rule='c 189:* rmw' \
+               -p 8554:8554 \
+               <the docker image you just built> \
+               /app/inference --input=uvc --fps=30
+    ``` 
+    where 
+    * --device=/dev/video0 allows the container to access your uvc camera video0.
+    * --input=uvc is the parameter for inference app which will then use uvc camera as input source.
+    * --fps=30 specifies the input fps from the uvc camera.
+1. Once it's started, you will see the inference results and the video stream with overlay bounding boxes and labels.
+
+---
+**NOTE**
+
+UVC camera is only enabled in object detection model as an example.
+Please follow the sample codes to enable more models if needed.
+
+---
+
+### Using pre-recorded video file as input source
+
+Pre-recorded video file as input source in also supported. Supported codec is H.264 and file format includes but not limited to *.mp4, *.mkv.
+1. Assume the docker image was ready.
+1. Prepare an H.264 pre-recorded video file and store in somewhere you want on the host, for example, /video/homes_00425.mkv.
+   You can download a sample video from https://abhilashavastorage.blob.core.windows.net/video/homes_00425.mkv
+1. Now run the Docker container like this:
+    ```
+    docker run --rm \
+               -v /dev/bus/usb:/dev/bus/usb \
+               -v /video:/video \
+               -it \
+               --device-cgroup-rule='c 189:* rmw' \
+               -p 8554:8554 \
+               <the docker image you just built> \
+               /app/inference --input=video:/video/homes_00425.mkv --fps=30
+    ``` 
+    where
+    * -v /video:/video is to bind the /video folder on your host to the folder with same name in container.
```

Comment:
Typo. `/video:/video` should be `video:/video`, right?

Reply:
Actually my video folder is located in root. That's why I wrote /video as the first one. So /video:/video is correct in my opinion.
Or did I miss anything?

Reply:
You say `--input=video:/video/blah` but then in the description you say `/video:/video`. If you don't think it is a typo, then I am misunderstanding it, and it doesn't really matter. Close this comment out or whatever.

Reply:
the _video:_ in --input is a static prefix, while the _/video_ in -v is a folder.
Maybe next time I should change the folder name to other name to better distinguish.
It's not a typo here.